### PR TITLE
Add payment options to owner profile

### PR DESCRIPTION
**Pull request by timomeh**
- Adds UI for managing the subscription

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -2,4 +2,5 @@
 BUILD_VARIANT=rcvr
 NEXT_PUBLIC_BUILD_VARIANT=$BUILD_VARIANT
 
-NEXT_PUBLIC_API_BASE=https://api.rcvr.app/
\ No newline at end of file
+NEXT_PUBLIC_API_BASE=https://api.rcvr.app/
+NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY=pk_test_Q2N3ljTzzBgxO4uOlsv5BS7u00f3eocyfC
```
Comment (bonflintstone) [inline .env:6]:
```suggestion
NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY=pk_live_4e9OH54b1NzmfES809cLedMu00F8m8gRka
```

File: .env.production [hunk 2]
```diff
@@ -0,0 +1 @@
+NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY=pk_test_Q2N3ljTzzBgxO4uOlsv5BS7u00f3eocyfC
```
Comment (bonflintstone) [inline .env.production:1]:
```suggestion
NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY=pk_live_4e9OH54b1NzmfES809cLedMu00F8m8gRka
```
Not sure which of the two it should be, just commit the auggestion wher you see fit :)

File: .env (hunk 1 above)
Comment (timomeh) [reply .env:6]: Can we still use the test key for testing? Or is it invalid now?

Comment (timomeh) [reply .env:6]: We'll leave this here so we can use the test key in development

**Pull request merged**